### PR TITLE
Add DCT signing step

### PR DESCRIPTION
**Pull request by zdhughes**
*Description of changes:*

This PR adds a DCT signing step to our CI process.

### CODE REVIEW

File: .github/workflows/deploy.yaml [hunk 1]
```diff
@@ -10,42 +10,47 @@ jobs:
...
         run: |
           aws ecr get-login-password --region=us-east-1 | docker login --username AWS --password-stdin ${{ secrets.ECR_REPOSITORY }}
-      -
-        name: Build and push
+      - name: Build and push
         uses: docker/build-push-action@v2
         with:
           context: .
-          push: true
+          load: true
```
Comment (emyann) [inline .github/workflows/deploy.yaml:37]: What's is the difference between `load` and `push` ?

**Review by emyann — APPROVED** [review]
LGTM! Thanks for setting DCT up!

File: .github/workflows/deploy.yaml (hunk 1 above)
Comment (zdhughes) [reply .github/workflows/deploy.yaml:37]: `load` indicates that the action should export the Docker image to make it available to other steps. `push` indicates that the action should push the specified tags. 

This action wasn't picking up the DCT variables it needed to in order for us to sign this, so we're now using this step to build, and doing the actual push in the following step.

Comment (emyann) [reply .github/workflows/deploy.yaml:37]: Oh nice, thanks for the insight!

Comment (zdhughes) [reply .github/workflows/deploy.yaml:37]: Any time!

**Pull request merged**